src/chart/main/style.cpp

```diff
@@ -462,25 +462,25 @@ Chart Chart::def()
 					.numberScale = ::Text::NumberScale{}
 				},
 			},
-			.footer = {
+			.caption = {
 				Padding
 				{
-					.paddingTop = Gfx::Length::Emphemeral(0),
-					.paddingRight = Gfx::Length::Emphemeral(10.0 / 26.0),
-					.paddingBottom = Gfx::Length::Emphemeral(15.0 / 26.0),
-					.paddingLeft = Gfx::Length::Emphemeral(10.0 / 26.0)
+					.paddingTop = Gfx::Length::Relative(0),
```

Comment:
Can we use em unit here as well?

Reply:
https://github.com/vizzuhq/vizzu-lib/pull/349/commits/6f0ce793636482f2382f3f4bea825bbf80d69588